parlai/utils/distributed.py

```diff
@@ -229,21 +230,27 @@ def sync_object(data, max_size=16384):
     return data
 
 
-def check_synced_parameters(model):
+def sync_parameters(model: torch.nn.Module) -> bool:
     """
-    Check that all parameters across all workers are the same.
+    Sync all parameters across all workers are the same.
 
-    Always returns True, or raises an AssertionError if they are not
-    synchronized.
+    Always returns True, or raises an AssertionError if there was a failure.
 
-    :param torch.nn.Module model: A pytorch model.
-    :return: True
+    :param model: A pytorch model.
+    :return: always True
     """
     if not is_distributed():
         # if things aren't distributed, of course things are in sync
         return True
 
-    # compute the local norm:
+    # syncrhonize all the parameters
```

Comment:
nit: synchronize

Reply:
how do i manage to write any code correctly?